### PR TITLE
Add .xcf

### PR DESCRIPTION
**Pull request by tom-englert**
to support https://github.com/Fody/Fody/pull/598 as soon as this feature is released.

### CODE REVIEW

File: PropertyChanged.Fody/PropertyChanged.Fody.xcf [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<xs:complexType xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:attribute name="InjectOnPropertyNameChanged" type="xs:boolean">
+    <xs:annotation>
+      <xs:documentation>Used to control if the On_PropertyName_Changed feature is enable.</xs:documentation>
```
Comment (ltrzesniewski) [inline PropertyChanged.Fody/PropertyChanged.Fody.xcf]:
```suggestion
      <xs:documentation>Used to control if the On_PropertyName_Changed feature is enabled.</xs:documentation>
```

File: PropertyChanged.Fody/PropertyChanged.Fody.xcf [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    </xs:annotation>
+  </xs:attribute>
+  <xs:attribute name="CheckForEquality" type="xs:boolean" >
+    <xs:annotation>
+      <xs:documentation>Used to control if equality checks should be created. If false, equality checking will be disabled for the project.</xs:documentation>
+    </xs:annotation>
+  </xs:attribute>
+  <xs:attribute name="CheckForEqualityUsingBaseEquals" type="xs:boolean" >
+    <xs:annotation>
+      <xs:documentation>Used to control if equality checks should use the equals method resolved from the base class.</xs:documentation>
```
Comment (ltrzesniewski) [inline PropertyChanged.Fody/PropertyChanged.Fody.xcf]:
```suggestion
      <xs:documentation>Used to control if equality checks should use the Equals method resolved from the base class.</xs:documentation>
```

File: PropertyChanged.Fody/PropertyChanged.Fody.xcf [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    </xs:annotation>
+  </xs:attribute>
+  <xs:attribute name="UseStaticEqualsFromBase" type="xs:boolean" >
+    <xs:annotation>
+      <xs:documentation>Used to control if equality checks should use the static equals method resolved from the base class.</xs:documentation>
```
Comment (ltrzesniewski) [inline PropertyChanged.Fody/PropertyChanged.Fody.xcf]:
```suggestion
      <xs:documentation>Used to control if equality checks should use the static Equals method resolved from the base class.</xs:documentation>
```

File: PropertyChanged.Fody/PropertyChanged.Fody.xcf [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    </xs:annotation>
+  </xs:attribute>
+  <xs:attribute name="EventInvokerNames" type="xs:string" >
+    <xs:annotation>
+      <xs:documentation>Used to change the name of the method that fires the notify event. This is a string that accepts multiple values in a comma separated form.</xs:documentation>
+    </xs:annotation>
+  </xs:attribute>
+  <xs:attribute name="CheckForEquality" type="xs:boolean" >
+    <xs:annotation>
+      <xs:documentation>Used to control if equality checks should be created. If false, equality checking will be disabled for the project.</xs:documentation>
```
Comment (ltrzesniewski) [inline PropertyChanged.Fody/PropertyChanged.Fody.xcf]:
```suggestion
      <xs:documentation>Used to control if equality checks should be inserted. If false, equality checking will be disabled for the project.</xs:documentation>
```

**Comment by SimonCropp** [comment]
thanks for this. it is now deployed

**Pull request merged**